modules/icinga/manifests/config/slack.pp

```diff
@@ -2,6 +2,6 @@
 class icinga::config::slack {
   package {'icinga-slack-webhook':
     ensure   => '2.0.0',
-    provider => 'pip',
+    provider => pip3,
```

Comment:
Do you need this quoted?

Reply:
I'm unsure, but I saw other examples of it not being quoted [1], so I'm hoping it'll work.

1: https://github.com/alphagov/govuk-puppet/blob/be023ee704b4850d9ccd1e64e9f5ae548b522a40/modules/govuk_postgresql/manifests/wal_e/package.pp#L18